indico/web/client/js/react/components/DatePickerCalendar.jsx

```diff
@@ -0,0 +1,78 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2024 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import {Translate} from 'indico/react/i18n';
+import {range} from 'indico/utils/iterable';
+
+const NUM_DAYS_PER_WEEK = 7;
+const NUM_CALENDAR_CELLS = 42; // 6 weeks x 7 days
+
+function DatePickerCalendarGrid({includeMonthHeader = false}) {
+  return (
+    <ind-date-grid data-grid>
+      {includeMonthHeader ? <div className="month-label" data-month-label /> : null}
+      <div className="weekdays" data-weekday-labels>
+        {Array.from(range(NUM_DAYS_PER_WEEK)).map(i => (
```

Comment:
You can probably use `_.range()` for this

Reply:
Tbh, that code is mostly there to appease the linter. Otherwise it's completely unnecessary. Using a 3rd party library for that one-liner honestly sounds like adding insult to injury.

Reply:
it's a library we're already using for many other things though (and you did have to implement `range` yourself first... ;))

Reply:
Actually, there's also a way to do it with just standard functionality and without a custom function:

`Array(rows).fill().map((_, i) => ...)`

Personally I don't have a strong opinion on this one though.

Reply:
That's exactly what I would use. However, that trips the linter which doesn't really understand the context.